internal/runtime/runtime.go

```diff
@@ -145,9 +181,6 @@ func (gr *GolangRuntime) ProcessMessage(edgexcontext *appcontext.Context, envelo
 	return nil
 }
 
-// SetTransforms is thread safe to set transforms
-func (gr *GolangRuntime) SetTransforms(transforms []appcontext.AppFunction) {
-	gr.isBusyCopying.Lock()
-	gr.transforms = transforms
-	gr.isBusyCopying.Unlock()
+func (gr *GolangRuntime) StartStoreAndForward(appCtx context.Context, enabledCtx context.Context, serviceKey string, config *common.ConfigurationStruct, edgeXClients common.EdgeXClients) {
```

Comment:
Break line -- it's too long.  Maybe to this?

```
func (sf *storeForwardInfo) startStoreAndForwardRetryLoop(
	appCtx context.Context,
	enabledCtx context.Context,
	serviceKey string,
	config *common.ConfigurationStruct,
	edgeXClients common.EdgeXClients) {
```

There may be other lines in this commit that exceed reasonable (120 character) line length that should be folded.  Only calling out this instance.